src/components/v5/common/Table/partials/BurgerMenu.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import { BurgerMenuProps } from '../types';
+import Card from '~v5/shared/Card';
+
+const displayName = 'v5.common.Table.partials.BurgerMenu';
+
+const BurgerMenu: FC<BurgerMenuProps> = ({
```

Comment:
I think that you can remove this component because @CzarekDryl is working on it in another PR. You can add a TODO in a place where you're using it

Reply:
I'll add TODO to remove it after @CzarekDryl will be merged.